setup.py

```diff
@@ -24,8 +24,6 @@
 
 root_dir = os.path.dirname(os.path.abspath(__file__))
 
-project_name = os.getenv("PROJECT_NAME", "gstaichi")
```

Comment:
since we are setting project name in pyproject.toml, pointless to try to set it here too.